projects/go-lib/src/lib/components/go-timepicker/go-timepicker.component.ts

```diff
@@ -0,0 +1,94 @@
+import { Component, ElementRef, Input, OnInit, ViewChild } from '@angular/core';
+import { FormControl } from '@angular/forms';
+import { generateId } from '../../utilities/form.utils';
+
+@Component({
+  selector: 'go-timepicker',
+  styleUrls: ['./go-timepicker.component.scss'],
+  templateUrl: './go-timepicker.component.html',
+})
+
+export class GoTimepickerComponent implements OnInit {
+  displayAbove: boolean = false;
+  displayFromRight: boolean = true;
+  timeOpen: boolean = false;
+  id: string;
+  openTimeValue: string;
+
+  @Input() key: string;
+  @Input() label: string;
+  @Input() control: FormControl;
+  @Input() hints: string[];
+  @Input() placeholder: string = '';
+  @Input() theme: string = 'light';
+  @Input() appendToContent: boolean = false;
+
+  selectedTime: string;
+
+  @ViewChild('datepickerInput', { static: true }) datepickerInput: ElementRef;
+
+  ngOnInit(): void {
+    this.id = this.key || generateId(this.label, 'timepicker');
+    this.selectedTime = this.changeTimeFormat(this.control.value);
+  }
+
+  public changeTimeFormat(timeString: string): string {
```

Comment:
Can this method be changed to private? It doesn't look like we're using this method outside of this component.